include/obmc_console.hpp

```diff
@@ -118,29 +119,12 @@ inline void requestRoutes(App& app)
     BMCWEB_ROUTE(app, "/console0")
         .privileges({{"ConfigureManager"}})
         .websocket()
-        .onopen(
-            [](crow::websocket::Connection& conn) {
-        BMCWEB_LOG_DEBUG << "Connection " << &conn << " opened";
-
-        sessions.insert(&conn);
-        if (hostSocket == nullptr)
```

Comment:
Why remove the hostSocket==nullptr check?

Reply:
The check is done after the user validation at the line 169.

```
            const ::redfish::Privileges requiredPrivileges{"ConfigureManager"};
            if (!userPrivileges.isSupersetOf(requiredPrivileges))
            {
...
                return;
            }
            sessions.insert(&conn);
L169>    if (hostSocket == nullptr)
            {
                const std::string consoleName("\0obmc-console", 13);
                boost::asio::local::stream_protocol::endpoint ep(consoleName);
                hostSocket = std::make_unique<
                    boost::asio::local::stream_protocol::socket>(
                    conn.getIoContext());
                hostSocket->async_connect(ep, connectHandler);
            }
```